Modified toast

nameerakhter opened the pull request:
### Description:
This PR addresses the issue #12 

### Issue Fixed:
issue #12 


### Please check all the relevant boxes:
- [❌] Does this PR break UI anywhere?
- [✅ ] Have you tested the changes thoroughly before creating this PR?
- [❌ ] Have you pushed any sensitive information along with the changes?
- [❌] Is this issue related to any other issue?
- [❌] Does your change generate any new warnings?
- [❌] Have you added screenshots with the PR?

nameerakhter commented:
@poswalsameer added some basic styles for now for toast messages let me know if these are OK or we can use something like react-toastify later that will give us a lot more customization over toast messages

poswalsameer commented on client/campusburn/src/app/login/page.tsx:
```diff
@@ -58,12 +60,17 @@ export default function Page() {
         else if( loginResponse.status === 200 ){
             console.log("Response after login: ", loginResponse);
-
+            toast({
```
This toast is not required on login successful, as we are already pushing the user to a new route.

nameerakhter replied:
Sure I'll update it

poswalsameer commented:
> @poswalsameer added some basic styles for now for toast messages let me know if these are OK or we can use something like react-toastify later that will give us a lot more customization over toast messages

ShadCN toasts are fine, if you want you can use [Sonner](https://ui.shadcn.com/docs/components/sonner) too.
Also, can you please attach the screenshots of the changes. That would help a lot.

nameerakhter commented:
@poswalsameer updated the changes as you suggested removed toast message on successful login

also here are some screenshots for the updated toast messages

updated success toast
![Screenshot 2024-12-14 230306](https://github.com/user-attachments/assets/8817cd45-6ad3-40fe-95e6-9fef0c6f0e10)
updated error toast

![Screenshot 2024-12-14 230416](https://github.com/user-attachments/assets/fb0532ef-563c-4fef-b1ac-2e259236ac5a)

poswalsameer commented:
> @poswalsameer updated the changes as you suggested removed toast message on successful login
> 
> also here are some screenshots for the updated toast messages
> 
> updated success toast ![Screenshot 2024-12-14 230306](https://private-user-images.githubusercontent.com/120779958/395807951-8817cd45-6ad3-40fe-95e6-9fef0c6f0e10.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyNTM1NzYsIm5iZiI6MTczNDI1MzI3NiwicGF0aCI6Ii8xMjA3Nzk5NTgvMzk1ODA3OTUxLTg4MTdjZDQ1LTZhZDMtNDBmZS05NWU2LTlmZWYwYzZmMGUxMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQwOTAxMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMDJjYzk5YTcwNTM5MWRhNjAyZmExZGQ1NzQ0NWI2NjY3NTZiYmZkMGU3ZWZmMjJmMmNjMGM3NDdlYjY3MDRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.nyVq-HCXdtp8MZxMxtKmPhnDzokrqrL-evFR_W98WhU) updated error toast
> 
> ![Screenshot 2024-12-14 230416](https://private-user-images.githubusercontent.com/120779958/395807954-fb0532ef-563c-4fef-b1ac-2e259236ac5a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyNTM1NzYsIm5iZiI6MTczNDI1MzI3NiwicGF0aCI6Ii8xMjA3Nzk5NTgvMzk1ODA3OTU0LWZiMDUzMmVmLTU2M2MtNGZlZi1iMWFjLTJlMjU5MjM2YWM1YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQwOTAxMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03N2YwOTBiOGQxYTM1Y2FkMzQxYTJkMGQzNmZlMTkzOWE1MGM2NjAwYWEyY2M3YzM5NTBmYTcxMzc2MTdhYjI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.JU-Iwuzql3LJQyGvYWsYgRsxELTF-fiSB2nxyDp_B8o)

Can you prepare a design like the one attached below? Basically, we need to have minimal toasts with a gradient and transparent background.

![Screenshot 2024-12-15 143242](https://github.com/user-attachments/assets/b2d5ce8e-f6a8-4f9b-9437-1b8e4f60c9c0)

nameerakhter commented:
> > @poswalsameer updated the changes as you suggested removed toast message on successful login
> > also here are some screenshots for the updated toast messages
> > updated success toast ![Screenshot 2024-12-14 230306](https://private-user-images.githubusercontent.com/120779958/395807951-8817cd45-6ad3-40fe-95e6-9fef0c6f0e10.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyNTM1NzYsIm5iZiI6MTczNDI1MzI3NiwicGF0aCI6Ii8xMjA3Nzk5NTgvMzk1ODA3OTUxLTg4MTdjZDQ1LTZhZDMtNDBmZS05NWU2LTlmZWYwYzZmMGUxMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQwOTAxMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMDJjYzk5YTcwNTM5MWRhNjAyZmExZGQ1NzQ0NWI2NjY3NTZiYmZkMGU3ZWZmMjJmMmNjMGM3NDdlYjY3MDRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.nyVq-HCXdtp8MZxMxtKmPhnDzokrqrL-evFR_W98WhU) updated error toast
> > ![Screenshot 2024-12-14 230416](https://private-user-images.githubusercontent.com/120779958/395807954-fb0532ef-563c-4fef-b1ac-2e259236ac5a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyNTM1NzYsIm5iZiI6MTczNDI1MzI3NiwicGF0aCI6Ii8xMjA3Nzk5NTgvMzk1ODA3OTU0LWZiMDUzMmVmLTU2M2MtNGZlZi1iMWFjLTJlMjU5MjM2YWM1YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQwOTAxMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03N2YwOTBiOGQxYTM1Y2FkMzQxYTJkMGQzNmZlMTkzOWE1MGM2NjAwYWEyY2M3YzM5NTBmYTcxMzc2MTdhYjI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.JU-Iwuzql3LJQyGvYWsYgRsxELTF-fiSB2nxyDp_B8o)
> 
> Can you prepare a design like the one attached below? Basically, we need to have minimal toasts with a gradient and transparent background.
> 
> ![Screenshot 2024-12-15 143242](https://private-user-images.githubusercontent.com/106386145/395844124-b2d5ce8e-f6a8-4f9b-9437-1b8e4f60c9c0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyNTM4MTIsIm5iZiI6MTczNDI1MzUxMiwicGF0aCI6Ii8xMDYzODYxNDUvMzk1ODQ0MTI0LWIyZDVjZThlLWY2YTgtNGY5Yi05NDM3LTFiOGU0ZjYwYzljMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQwOTA1MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMWQ1MDRlNmE1ZTQwMDhkZThiMjE2ZWEzNDcyNTMyNWRmMWRmZTNkNzRmMjZmNWQyMDZkYWQ2ODMxM2Q1MDk4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oU28kxQd0yZ_OnY8PzuDfb1I-cMHrocBqNTpIYiTPps)

Sure I'll design it and update the PR also let me know if you want the same layout as well like on top you want to show a title of toast and then you want to add a description as well for toast

nameerakhter commented:
@poswalsameer 

updated the Toast messages here are the screenshots let me know if these work

![Screenshot 2024-12-15 155225](https://github.com/user-attachments/assets/09de7550-4e04-429b-afa7-8fa041239370)

![Screenshot 2024-12-15 155151](https://github.com/user-attachments/assets/bc6836ac-7a7a-43f8-a7ac-bff7973b27f2)

poswalsameer commented:
> @poswalsameer
> 
> updated the Toast messages here are the screenshots I didn't exactly tried to replicate the design because then I would have to write custom CSS just tailwind might not be enough, let me know what you think
> 
> ![Screenshot 2024-12-15 155225](https://private-user-images.githubusercontent.com/120779958/395854356-09de7550-4e04-429b-afa7-8fa041239370.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyNzkzMTcsIm5iZiI6MTczNDI3OTAxNywicGF0aCI6Ii8xMjA3Nzk5NTgvMzk1ODU0MzU2LTA5ZGU3NTUwLTRlMDQtNDI5Yi1hZmE3LThmYTA0MTIzOTM3MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQxNjEwMTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZWIxYmVjODIyZjMxNTdiYzZiZmY4OTRlNTUwYTRhZjRhZDhhM2JjODE5Yzg5Mjk1ZGYzZTRlYmIxMTkwMWY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ec0CfzGbW2xiObtQLV-XlGPAYc5s_r9FjpolNaw6vmY)
> 
> ![Screenshot 2024-12-15 155151](https://private-user-images.githubusercontent.com/120779958/395854360-bc6836ac-7a7a-43f8-a7ac-bff7973b27f2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQyNzkzMTcsIm5iZiI6MTczNDI3OTAxNywicGF0aCI6Ii8xMjA3Nzk5NTgvMzk1ODU0MzYwLWJjNjgzNmFjLTdhN2EtNDNmOC1hN2FjLWJmZjc5NzNiMjdmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNVQxNjEwMTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYzQ0ZmQ0ZTI2ODE4Mjk1MjYzOTMxOTQzYmQ4ZWJkMGVlYzg5NTViN2IwYmFkMTZjNzM1OGQ3YjNmYmY1NzAyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.nBHGkDR-RDshZg2H8R5G9_cE2DDcqsRpDxWwHeCt5YA)

Cool, merging this now. Thanks buddy!
